storage/innobase/fil/fil0fil.cc

```diff
@@ -188,19 +188,19 @@ bool
 fil_validate_skip(void)
 /*===================*/
 {
-	/** The fil_validate() call skip counter. Use a signed type
-	because of the race condition below. */
+	/** The fil_validate() call skip counter. */
 	static int fil_validate_count = FIL_VALIDATE_SKIP;
 
-	/* There is a race condition below, but it does not matter,
-	because this call is only for heuristic purposes. We want to
-	reduce the call frequency of the costly fil_validate() check
-	in debug builds. */
-	if (--fil_validate_count > 0) {
+	/* We want to reduce the call frequency of the costly fil_validate()
+	 * check in debug builds. */
```

Comment:
Could you remove star here and align similarly to original comment?

Reply:
Like this:
```
	/* We want to reduce the call frequency of the costly fil_validate()
	check in debug builds. */
```

Reply:
Fixed.